geppetto-ui/src/list-viewer/ListViewer.js

```diff
@@ -348,7 +348,7 @@ export default class ListViewer extends React.Component {
       <div className="listviewer-container">
         <Filter />
         <Table />
-        <Pagination />
+        { this.props.showPagination === false ? null : <Pagination /> }
```

Comment:
the prop showPagination needs to be documented in the [propTypes ](https://github.com/openworm/geppetto-client/blob/f394c1b8060e0f7b747aad74d1106244b789193d/geppetto-ui/src/list-viewer/ListViewer.js#L395) and added to the showcase. Same would be needed for the `layout `property, which is another way to implement this behaviour.